setupTests.ts

```diff
@@ -39,7 +39,7 @@ export const buildTestSpectralWithAsyncApiRule = async (ruleName: string): Promi
   expect(Object.keys(s.rules)).toEqual([ruleName]);
 
   const rule = s.rules[ruleName];
-  expect(rule.recommended).not.toBe(false);
```

Comment:
Bringing back memories! :)

Reply:
This looks like it's going backwards. I was pointing this out as a mistake, sorry for not being clear, We removed enabled so why does this test talk about enabled instead of recommended?

Reply:
`enabled` is internal, nothing exposed to end user.
Basically, it's a combination of recommended + severity, but it's something end user will be unaware of.
This is a check we had to do in the lining process to verify whether a given rule should be executed or not, so I simply extracted it to a single spot for our own convenience.

Reply:
@philsturgeon I think @P0lip's change is sound. `recommended` is a rule property in the context of the ruleset. Once loaded, it's of little use. However, `enabled` is needed from the rule engine standpoint.

And that's one of thing I do like with this PR. It disambiguates a RulesetRule from a loaded one.

FWIW, I'd have taken this one step further and kept the initial `severity` level from the ruleset (without turning it off when not `recommended`), which would help on the test standpoint and avoid code like https://github.com/stoplightio/spectral/blob/f8b683b044698f59640e54603331b23a6ce96d80/src/rulesets/__tests__/reader.jest.test.ts#L217